### PR TITLE
examples(building-dashboard-supabase): update samples

### PR DESCRIPTION
**Pull request by Shadid12**
**Check List**
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required

**Issue Reference this PR resolves**

[For example #12]

**Description of Changes Made (if issue reference is not provided)**

[Description goes here]

### CODE REVIEW

File: examples/supabase/src/components/BarChart.js [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      <YAxis />
+      <CartesianGrid />
+      {children}
+      <Legend />
+      <Tooltip />
+    </ChartComponent>
+  </ResponsiveContainer>
+);
+
+const cubejsApi = cubejs(
```
Comment (sashasushko) [inline examples/supabase/src/components/BarChart.js:29]: Can we use `CubeProvider` and initialize API once in `App.js` or `index.js` instead?

Comment (sashasushko) [reply examples/supabase/src/components/BarChart.js:29]: Here is example - https://cube.dev/docs/@cubejs-client-react

Comment (Shadid12) [reply examples/supabase/src/components/BarChart.js]: updated in new commit

**Review by adnanrahic — APPROVED** [review]
Absolutely beautiful. 👌

**Review by sashasushko — APPROVED** [review]

**Pull request merged**